content/en/dashboards/guide/how-to-use-terraform-to-restrict-dashboard-edit.md

```diff
@@ -7,36 +7,54 @@ aliases:
 ---
 
 
-## Introduction
+## Background
 
-Previously when you wanted to restrict editing of dashboards created and managed by [Terraform][1], you would use the `is_read_only` attribute to define that editing the dashboard is restricted to the creator or users with the Access Management (`user_access_manage`) permission in your organization. With the introduction of `restricted_roles`, you can list specific roles that can edit this dashboard within your organization.
+Previously, [granular access controls][1] for dashboards were managed through the `is_read_only` and `restricted_roles` attributes. These attributes are being deprecated, and it is advised to use [Restriction Policies][2] to configure access instead.
 
-## Restricting a dashboard
+<div class="alert alert-warning">
+Terraform Restriction Policies for dashboards are currently in private beta, please reach out to your Datadog contact or support to enable the feature.
+</div>
```

Comment:
Re: warnings
* "Currently" seems appropriate since it's not going to be in private beta forever
* I don't see why it's better to drop the please, happy to do so if anyone else feels like it